src/com/oltpbenchmark/api/Loader.java

```diff
@@ -173,7 +173,7 @@ public void unload(Catalog catalog) throws SQLException {
         Statement st = conn.createStatement();
         for (Table catalog_tbl : catalog.getTables()) {
             LOG.debug(String.format("Deleting data from %s.%s", workConf.getDBName(), catalog_tbl.getName()));
-            String sql = "DELETE FROM " + catalog_tbl.getEscapedName();
+            String sql = "truncate " + catalog_tbl.getEscapedName();
```

Comment:
Is this comdb2 specific?

Reply:
There were a number of issues in trying to get TPCH play well with ComDB2 - Most of the changes in the Oltpbench source were our attempts(and suggestions from ComDB2 devs) to get it to work. @QinyuHu and I will revert these changes soon since TPC-H didnt end up working.

Reply:
We will revert the changes - they only change "DELETE FROM" to "truncate" for speed up reason. Also System.out was used for debug purpose so we will also revert them.